lib/beeswaxapi/request.rb

```diff
@@ -59,7 +59,12 @@ def request_for(**opts)
       end
 
       if opts.has_key? :body_params
-        opts[:body] = opts.delete(:body_params)
+        body = opts.delete(:body_params)
+        opts[:body] = Yajl.dump(body)
```

Comment:
ah.. so initially this gem wasn't converting the body to a json?

Reply:
It did, but all possible attributes in `body_params` and in case of sending a file it broke the process.